core/services/feeds/rpc_handlers.go

```diff
@@ -28,10 +28,11 @@ func (h *RPCHandlers) ProposeJob(ctx context.Context, req *pb.ProposeJobRequest)
 	}
 
 	jp := &JobProposal{
-		Spec:           req.Spec,
+		Spec:           req.GetSpec(),
 		Status:         JobProposalStatusPending,
 		FeedsManagerID: h.feedsManagerID,
 		RemoteUUID:     remoteUUID,
+		Multiaddrs:     req.GetMultiaddrs(),
```

Comment:
These multiaddrs only make sense for OCR jobs, any sense in rejecting a proposal upfront if we get a FM jobs with multiaddrs @jkongie @begmaroman ?

Reply:
Agree. It is done on the service layer.